CHANGELOG.md

```diff
@@ -21,7 +24,7 @@
 
 [Full changelog](https://github.com/mozilla/glean/compare/v26.0.0...v27.0.0)
 
-* General:
+* General
```

Comment:
Looks like this is typically used with the `:` in most other places...

Reply:
Issue in master, I'd rather fix it there at some point.